medasdigital/assetlist.json

```diff
@@ -0,0 +1,29 @@
+{
+    "$schema": "../../assetlist.schema.json",
+    "chain_name": "medasdigital-1",
+    "assets": [
+      {
+        "description": "The native token of Medas Digital Network",
+        "denom_units": [
+          {
+            "denom": "umedas",
+            "exponent": 0,
+            "aliases": []
+          },
+          {
+            "denom": "medas",
+            "exponent": 6,
+            "aliases": []
+          }
+        ],
+        "base": "umedas",
+        "name": "Medas Digital",
+        "display": "medas",
+        "symbol": "MEDAS",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/medasdigital/images/medas.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/medasdigital/images/medas.svg"
```

Comment:
this svg image appears blank for me. Could there be an issue with this svg?

Reply:
Good Morning Jeremy, thank you for your assistance. I exchanged the svg.

Reply:
Good day, thanks for looking into it. But I'm Sorry the image does not match the png. the Png correctly has transparency, while the svg has grey where is should be transparent. Could you please correct?

Reply:
Hi, recreated SVG. In github it now looks good